### ISSUE TITLE
Huge Log Files

### ISSUE THREAD

**Issue by JohnPremKumar** [judge0/api#165]
Log size grows indefinitely inside the /var/lib/docker/overlay2/ folder.

```
/var/lib/docker/overlay2/<hash>/merged/usr/src/api/log

13.4 GiB [##########]  production.log
```

```
/var/lib/docker/overlay2/<hash>/diff/usr/src/api/log

13.4 GiB [##########]  production.log
```

Is there is any way to limit this behavior?. Also is it safe to delete these log files directly?

**Comment by abhishekv24** [comment]
@hermanzdosilovic I am facing the same problem with api v1.5.0. I tried limiting the log files size by configuring the logging options in the docker-compose.yml file with no success.

Later I also referred to the new commit @https://github.com/judge0/api/commit/435c7d792babfda8564dd432d31d4edc5a694ab9 no success with that too.

**Comment by hermanzdosilovic** [comment]
Hi @JohnPremKumar and @abhishekv24,

it is safe to delete those files, but you cloud also add `RAILS_LOG_TO_STDOUT=true` to judge0-api.conf.

This issue should be resolved with the Judge0 API v1.6.0 with [this commit](https://github.com/judge0/api/commit/37d222bb47be140f3553b0cd448b8679c8578a93).

If you can, please consider using the latest version [v1.6.0](https://github.com/judge0/api/releases/tag/v1.6.0).

**Comment by JohnPremKumar** [comment]
Thanks for the update @hermanzdosilovic, I deleted the production.log inside the merged/..... folder directly and restarted the docker service to reclaim the unused space.

Everything worked without any issues.
